src/index.test.ts

```
@@ -837,269 +1505,793 @@ describe('checkbox-search prompt', () => {
+          value: originalRows,
+          configurable: true,
+        });
+      } else {
+        // Remove the property if it didn't exist originally
+        delete (process.stdout as any).rows;
+      }
+    });
+
+    it('should return fallback when terminal height is not available', () => {
+      // Mock process.stdout.rows as undefined
+      vi.spyOn(process.stdout, 'rows', 'get').mockReturnValue(undefined as any);
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)